shell/common/platform_util_linux.cc

```diff
@@ -122,6 +123,14 @@ bool XDGUtil(const std::vector<std::string>& argv,
   // bring up a new terminal if necessary.  See "man mailcap".
   options.environment["MM_NOTTTY"] = "1";
 
+  // If the user set a GDK_BACKEND value of their own, use that,
+  // otherwise unset it. Setting values in EnvironmentMap to an empty-string
+  // will make sure that they get unset from the environment via
+  // AlterEnvironment().
```

Comment:
The comment should mention that we are doing this because Chromium is setting `GDK_BACKEND` during GTK initialization.